### ISSUE TITLE
How gluon make aab file?

### ISSUE THREAD

**Issue by ctoabidmaqbool** [gluonhq/gluonfx-maven-plugin#449]
Hi! Dev, I am still using old `javafx-mobile-plugins`. I can successfully create APK file for **androidTarget 31**, without any issues.

So now, how can I make **AAB** file ? is there is simple solution to make AAB file using **old plugin**, or **convert APK To AAB** file directly. 

I have tried several ways to convert APK To AAB but some file in res folder becomes missing due to which AAB file not working properly after uploading to google playstore.

e.g.
https://play.google.com/store/apps/details?id=com.techbajao.apktoaab&hl=en&gl=US
https://community.niotron.com/t/convert-apk-to-aab-file/9684

Basically when APK is converted to AAB all others files (highlighted files as shown) are deleted
![image](https://user-images.githubusercontent.com/10474725/199163323-b068bb2b-e001-4675-83fe-ef99ebc36f2d.png)

Any simple ways to achieve AAB file with APK?

**Comment by ctoabidmaqbool** [comment]
Another issue or question I have.

How can I make resourses.arsc file? As when Apk file is build using old javafx-mobile-plugin. no such files is made that's cause missing recourse files when ApkTool applied.

**Comment by ctoabidmaqbool** [comment]
https://github.com/iBotPeaches/Apktool/issues/2919

Related, to this issue,

How resources files are parse to android native directories e.g. res/raw or specific packages?

**Comment by ctoabidmaqbool** [comment]
What is resourses.arsc file, how gluon mobile uses it? what is purpose of this file in aab or apk file?

**Comment by RenateUSB** [comment]
resources.arsc holds all resources that aren't .xml or .png or res/raw
In short, it hold values (which includes styles).
Easy way to see: List your .apk as a .zip file.
Any resource that doesn't have its own file is in resources.arsc

**Comment by ctoabidmaqbool** [comment]
Okay! I have found-out the solution of the main problem.

In short If I put files in android specific `res` folder using old `javamobile-plugin` all files in correct [folder structure](https://developer.android.com/guide/topics/resources/providing-resources) then all files are correctly included in `resources.arsc` file

e.g. gradle_project/src/android/res
![image](https://user-images.githubusercontent.com/10474725/200153994-876884fc-0395-4465-b6fd-4656ce2bf2f9.png)

e.g. gradle_project/src/main/resources
![image](https://user-images.githubusercontent.com/10474725/200154171-64d2bbb4-447b-4e7a-aef5-b2b890b99f4e.png)

I want to put file in common directory which should be used in main, desktop and andorid all, how should i do ?

**Comment by ctoabidmaqbool** [comment]
The resources under `src/main/resources` are copied to `src/android/res`. But why they are not technical counted as android native resources and included in resouces.arsc file?

**Comment by ctoabidmaqbool** [comment]
files under `\src\main\resources\res\raw` are copied to `src\android\res\raw` but 

but these  `\src\main\resources\res\raw`  are not included to `resources.arsc` files however others are included.

**Comment by ctoabidmaqbool** [comment]
gluonhq community is not much helpful in old plugins support!

**Comment by mipastgt** [comment]
Maybe that's because you are the only one who is still riding this dead horse and all others have already upgraded to the currently supported plugins :-).

**Comment by ctoabidmaqbool** [comment]
@mipastgt No this is not a case! I think many are switching to android studio like codding due to poor support.

If latest plugin was 100% usable then I will also switch to latest plugin. I have tried many times. And again trying time after time if it becomes usable for me too.

For android project I am still using jdk-8 with javafx mobile plugin, however for desktop and web using latest jdk-17.

Latest gluon plugin is using grallvm which itself is not 100% usable. Say how you can use [FlywayDB](https://flywaydb.org/) with latest plugin and grallvm. Have you any idea.

However I have solve the problem by selft. e.g. using javafx-mobile-plugina and jdk-8 make final apk file for targetSdk 31 and uploaded to google playstore too, secussfully.
